package.json

```diff
@@ -30,34 +31,34 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "cliui": "^3.2.0",
-    "eslint-rule-documentation": "^1.0.0",
-    "path-is-absolute": "^1.0.1",
-    "which": "^1.2.8",
-    "window-size": "0.3.0",
-    "yargs": "^8.0.1"
+    "cliui": "^5.0.0",
```

Comment:
let's not upgrade things past major versions; that's far too breaking a change to be useful.

Reply:
Went to v4 as not to skip a major.

Reply:
let's keep it on v3.

Reply:
aha, ok